mopro-core/Cargo.toml

```diff
@@ -49,6 +53,7 @@ wasmer = { git = "https://github.com/oskarth/wasmer.git", rev = "09c7070" }
 toml = "0.8"
 serde = "1.0"
 serde_derive = "1.0"
+witness = { git = "https://github.com/philsippl/circom-witness-rs.git", features = ["build-native-witness"], optional = true}
```

Comment:
CI probably failing because you add a feature flag here that doesn't exist in that git repo.

If you added this feature flag in your fork, need to use forked version. If not, need to remove this